flytekit/core/python_auto_container.py

```diff
@@ -292,16 +292,18 @@ def name(self) -> str:
 
     @timeit("Load task")
     def load_task(self, loader_args: List[str]) -> PythonAutoContainerTask:
+        _, entity_name, *_ = loader_args
         import gzip
 
         import cloudpickle
 
         with gzip.open(PICKLE_FILE_PATH, "r") as f:
-            return cloudpickle.load(f)
+            entity_dict = cloudpickle.load(f)
```

Comment:
This means each task will load a bigger pickle object than it needs to and that each task will need more memory to load everything up.

But given how normal fast-register creates a tarball with all the task I am okay with this.